traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/edge/CacheRegister.java

```diff
@@ -140,14 +143,42 @@ public void setDeliveryServiceMatchers(final TreeSet<DeliveryServiceMatcher> mat
 		this.deliveryServiceMatchers = matchers;
 	}
 
+	private String getRequestName(final Request request) {
```

Comment:
`Request` has a `getHostname()` method -- does that not work here? It appears to be set for both DNS and HTTP requests, but I imagine for DNS there might be a trailing `.` to strip off before doing a map lookup.

Reply:
You're right, I could totally use the `getHostname()` method. Also, looks like it already strips out the trailing `.` for DNS requests.